tests/secureserver.pl

```diff
@@ -292,10 +296,13 @@ sub exit_signal_handler {
     $SIG{TERM} = \&exit_signal_handler;
     # stunnel configuration file
     if(open(my $stunconf, ">", "$conffile")) {
-        print $stunconf "CApath = $capath\n";
         print $stunconf "cert = $certfile\n";
         print $stunconf "debug = $loglevel\n";
         print $stunconf "socket = $socketopt\n";
+        if ($mtls) {
```

Comment:
```suggestion
        if($mtls) {
```
whitespace nit, to match rest of code

Reply:
done